controllers/argocd/applicationset.go

```diff
@@ -454,3 +470,45 @@ func setAppSetLabels(obj *metav1.ObjectMeta) {
 	obj.Labels["app.kubernetes.io/part-of"] = "argocd-applicationset"
 	obj.Labels["app.kubernetes.io/component"] = "controller"
 }
+
+//deleteApplicationSetResources ... Triggers reconcillation of application set resources,logic for deletion of applicationset is in reconcile methods this method triggers the cleanup of resources using the reconcilation logic as CR changes
```

Comment:
If please fix the below in comments.

1. Provide space after // 
2. remove ...
3. typo in reconciliation

May be
// deleteApplicationSetResources triggers the cleanup of application set resources.